types/sentinelmonitor.pp

```diff
@@ -0,0 +1,12 @@
+type Redis::SentinelMonitor = Hash[String,Struct[{
```

Comment:
It's slightly confusing that the type is singular (monitor) but the content is multiple.
```suggestion
type Redis::SentinelMonitors = Hash[String,Struct[{
```

Reply:
applied